storage/client/client.go

```diff
@@ -46,6 +47,17 @@ type StorageClient struct {
 	logger *log.Logger
 }
 
+func translateTokenType(tokenType evm.EVMTokenType) apiTypes.EvmTokenType {
+	switch tokenType {
+	case evm.EVMTokenTypeERC20:
+		return apiTypes.EvmTokenTypeERC20
```

Comment:
dang ol storage depending on api

Reply:
ugh now I think this is fine. it lets us have the list of enum values in the api spec for the price of one copy of the values